static/app/views/performance/newTraceDetails/traceSearch/traceSearchEvaluator.tsx

```diff
@@ -475,6 +484,26 @@ function resolveValueFromKey(
       // If the value can be accessed directly, do so,
       // else check if the key is an entity key, sanitize it and try direct access again.
       // @TODO support deep nested keys with dot notation
+      // console.log('here', token, key, value[key]);
```

Comment:
There's a couple commented out console.logs in this file

Reply:
I need to remove that. I was testing to try and understand why the code didnt work